reflex/components/base/error_boundary.py

```diff
@@ -0,0 +1,115 @@
+"""A React Error Boundary component that catches unhandled frontend exceptions."""
+
+from typing import Any, Dict, List, Union
+
+from reflex.compiler.compiler import _compile_component
+from reflex.components.component import Component
+from reflex.components.el import div, p
+from reflex.components.tags import Tag
+from reflex.constants import EventTriggers, Hooks
+from reflex.vars import Var
+
+
+class ErrorBoundary(Component):
+    """A React Error Boundary component that catches unhandled frontend exceptions."""
+
+    library = "react-error-boundary"
+    tag = "ErrorBoundary"
+
+    def _render(self, props: Union[Dict[str, Any], None] = None) -> Tag:
```

Comment:
it still works if we just defined the given props with defaults rather that overriding `_render` with almost the same logic as the parent. will push a commit here